src/main/java/seedu/command/interpreter/SeminarCommandInterpreter.java

```diff
@@ -62,7 +62,7 @@ public Command decideCommand(String commandDescription) throws PacException {
             break;
```

Comment:
Perhaps eventParser.parse(commandParameters) can be written before the case statements to avoid repetition?